internal/pkg/agent/application/actions/handlers/handler_action_application.go

```diff
@@ -48,6 +49,17 @@ func (h *AppAction) Handle(ctx context.Context, a fleetapi.Action, acker acker.A
 		return fmt.Errorf("invalid type, expected ActionApp and received %T", a)
 	}
 
+	// Validate action
+	validated, err := protection.ValidateAction(*action, h.coord.Protection().SignatureValidationKey, "")
```

Comment:
what is desired behavior when passing empty agent id
will it fail validation because id is not in `fleet action with agents` or will it be applied anyways?